File: DartProgramming/bin/m006_looping/Looping03.dart
//Looping 'do-while'
import 'dart:io';

void main() {
  //Arithmetic: 1, 2, 3, 4, ...
  print('Deret Arithmetic : ');
  stdout.write('Input n : ');
  int n = int.parse(stdin.readLineSync()!);

  int x = 1;
  do {
    stdout.write('${x++}, ');
  } while(x != n+1);
}

File: C_Programming/m008_looping/Looping03.c
//
// Created by Kelvin HT on 1/19/22.
//

#include <stdio.h>

int main() {
    // Arithmetic: 1, 2, 3, 4, 5, ...
    int n;
    printf("Input n : ");
    scanf("%d", &n);

    // Looping 'do-while'
    int x = 0;
    do {
        printf("%d ", (x + 1));
        x++;
    } while (x < n);
    printf("\n");
}
